package schema.type;

import "metadata.proto";

message Content{
	optional int32					version = 1[default = 0];
	required string					url = 2[default = ""];
	optional string					base = 3;
	optional bytes					content = 4;
	optional string					content_type = 5;
	optional net.type.Metadatas		metadata = 6;
}